web/src/typeahead_helper.ts

```diff
@@ -422,10 +421,16 @@ function retain_unique_language_aliases(matches: string[]): string[] {
     return unique_aliases;
 }
 
-export function sort_languages(matches: string[], query: string): string[] {
-    const results = typeahead.triage(query, matches, (x) => x, compare_language);
+export function sort_languages(matches: LanguagePillData[], query: string): LanguagePillData[] {
```

Comment:
This got a little ugly, but I essentially wanted to take the language strings out of the object and then recreate the object again. I should probably make a new line or two before the `return` statement to help it be easier to read.